src/monitor/src/main/java/ee/ria/xroad/monitor/CertificateInfoSensor.java

```diff
@@ -54,45 +56,42 @@
 
     // give signer some time to become available
     private static final FiniteDuration INITIAL_DELAY = Duration.create(10, TimeUnit.SECONDS);
-    private static final String JMX_HEADER = "ID\tISSUER\tSUBJECT\tNOT BEFORE\tNOT AFTER\tSTATUS";
+    private static final String JMX_HEADER = "SHA1HASH\t\t\t\t\t\t\tCERT TYPE\t\tNOT BEFORE\t\tNOT AFTER";
```

Comment:
this (formatting jmx messages with tabs, as different clients would probably show less pretty output) a bit ugly, I wonder if there would be some better way? but it is my fault....and should be good enough for this use case.